website/content/platform/development/developer-guidelines/architectural_considerations.md

```diff
@@ -97,12 +97,10 @@ The `Fetcher` class implementation is based on the [TET pattern](/platform/devel
 
 The OBBject class is a generic class in the OpenBB platform that represents a standardized object for handling and manipulating data fetched from various providers. It extends the `Tagged` class and uses Python's generics to allow flexibility in the type of results it can handle.
 
-The class includes several fields such as results, provider, warnings, chart, and extra, which respectively represent the fetched data, the data provider, any warnings generated during data fetching, an optional chart object for visualizing the data, and a dictionary for any additional information.
```

Comment:
Why removing the mention to `chart`? It is still a field of the `OBBject`

Reply:
Ooooops! My bad, I got carried away with my yeeting by the looks of it!